Library/Homebrew/rubocops/shell_cmd.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "rubocops/extend/formula"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      # This cop audits shell commands `system` and `Utils.popen`.
+      class ShellCmd < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          find_every_method_call_by_name(body_node, :system).each do |method|
+            next unless match = regex_match_group(parameters(method).first, /[^"' ]*(?:\s[^"' ]*)+/)
+
+            good_args = match[0].gsub(" ", "\", \"")
```

Comment:
https://ruby-doc.org/stdlib-2.5.1/libdoc/shellwords/rdoc/Shellwords.html#method-c-escape might be a good fit here.

Reply:
I'm not sure I agree. My understanding is that `escape` would turn something like `"foo bar"` into `"foo\ bar"`. What this line does is turn the original into `"foo", "bar"`. Maybe I'm missing a use case of `escape`

Reply:
Yeh, ignore me if you think it's a bad fit, then.